stl/test/wsh_wood/src/main.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+#include <vector>
+#include <string>
+
+int main() {
+    std::vector<char> data;
```

Comment:
Indent 2 spaces

Reply:
Thank you for your correction, I overlooked that the default 4 indentation in vscode does not comply with the project specifications.

Reply:
> Indent 2 spaces
Teacher, do I still need to review it after I have made the changes?